dockstore-webservice/src/main/java/io/dockstore/webservice/core/language_parsing/LanguageParsingRequest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright 2021 OICR and UCSC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package io.dockstore.webservice.core.language_parsing;
```

Comment:
Might as well not add new ones, think the google style disallows underscores like this 

> Package names are all lowercase, with consecutive words simply concatenated together (no underscores). For example, com.example.deepspace, not com.example.deepSpace or com.example.deep_space.

https://google.github.io/styleguide/javaguide.html

Reply:
I'll enable the rule and make it a warning (since many of our package names don't do this)